### PR TITLE
Prepare v2.0.0-beta.1

### PR DESCRIPTION
**Pull request by mbwhite**
Signed-off-by: Matthew B White <whitemat@uk.ibm.com>
Change-Id: I1e77b81ea9488047f1ba64f6ee222c054fc573dc

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Please visit the [contributors guide](http://hyperledger-fabric.readthedocs.io/e
 
 ## Folder Structure
 
-This repo is structured as a monorepo using [Rush](https://rushjs.io/). Why a monorepo? There are multiple npm modules that are published from this repo, including end to end tests, and tooling. It is siginficant easier to manager within one repo - and Rush has proven to provide excellent support for management of issues such as different dependency versions.
+This repo is structured as a monorepo using [Rush](https://rushjs.io/). Why a monorepo? There are multiple npm modules that are published from this repo, including end to end tests, and tooling. It is significant easier to manager within one repo - and Rush has proven to provide excellent support for management of issues such as different dependency versions.
```
Comment (awjh-ibm) [inline CONTRIBUTING.md]: Significantly easier to manage?

File: ci/azure-pipelines.yml [hunk 2]
```diff
@@ -58,8 +58,10 @@ stages:
         # output some basic information for reference
         - job: displayenv
           steps:
+            - template: templates/build-data.yaml
             - script: |
```
Comment (jt-nti) [inline ci/azure-pipelines.yml]: The template above does the env bit and displays the version already.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ Please visit the [contributors guide](http://hyperledger-fabric.readthedocs.io/e
 
 ## Folder Structure
 
-This repo is structured as a monorepo using [Rush](https://rushjs.io/). Why a monorepo? There are multiple npm modules that are published from this repo, including end to end tests, and tooling. It is siginficant easier to manager within one repo - and Rush has proven to provide excellent support for management of issues such as different dependency versions.
+This repo is structured as a monorepo using [Rush](https://rushjs.io/). Why a monorepo? There are multiple npm modules that are published from this repo, including end to end tests, and tooling. It is significantly easier to manager within one repo - and Rush has proven to provide excellent support for management of issues such as different dependency versions.
```
Comment (awjh-ibm) [inline CONTRIBUTING.md]: significantly easier to **manage**

File: tools/scripts/release.sh [hunk 4]
```diff
@@ -46,7 +46,7 @@ elif [[ ${NEW_SUFFIX} == 'snapshot' ]]; then
   echo new suffix is ${NEW_SUFFIX}
   CURRENT_VERSION=$(jq '.version' ${DIR}/package.json | sed -r "s/\"([0-9]?[0-9]\.[0-9]?[0-9]\.[0-9]?[0-9])(.)*/\1/")
   NEW_VERSION=$( ${DIR}/node_modules/.bin/semver -i ${CURRENT_VERSION} )-snapshot
-  NEW_TAG="unstable-1.4"
+  NEW_TAG="2.0.0-beta.1-1.4"
```
Comment (awjh-ibm) [inline tools/scripts/release.sh]: Guess this shouldn't still have a 1.4 on the end?

Comment (mbwhite) [reply tools/scripts/release.sh]: that script isn't used anymore

**Pull request merged**